site-user-management/layouts/api_user_deletion.tpl

```diff
@@ -0,0 +1,229 @@
+{% if editmode %}
+  <!DOCTYPE html>
+  <html class="editmode" lang="{{ page.language_code }}">
+    <head prefix="og: http://ogp.me/ns#">
+      <style>
+        .content {
+          font-size: 16px;
+          font-family: sans-serif;
+        }
+
+        td,
+        th {
+          border: 1px solid #dddddd;
+          padding: 8px;
+          text-align: left;
+        }
+
+        table {
+          table-layout: fixed;
+          border-collapse: collapse;
+        }
+
+        #home-button, 
+        .user-list, 
+        .search-user,
+        .add-user {
+          padding: 16px;
+        }
+      </style>
+
+      <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+
+      {% if editor_locale == 'et' %}
+        {% assign str_add_email = "Soovid lisada e-posti aadressi? " %}
+        {% assign str_add = "Lisa" %}
+        {% assign str_search_email = "Soovid otsida kasutajat? " %}
+        {% assign str_insert_email = "Sisesta e-posti aadress" %}
+        {% assign str_home = "Mine tagasi esilehele" %}
+        {% assign language = 'et' %}
+        {% assign str_title = "Parooliga kaitstud lehtede kasutajate kustutamine üle API" %}
+      {% else %}
+        {% assign str_add_email = "Add an e-mail address: " %}
+        {% assign str_add = "Add" %}
+        {% assign str_search_email = "Search for an e-mail: " %}
+        {% assign str_insert_email = "Insert an e-mail" %}
+        {% assign str_home = "Go back to front page" %}
+        {% assign language = 'en' %}
+        {% assign str_title = "Deleting users of password-protected pages over API" %}
```

Comment:
Korrektsem oleks "Manage users of password-protected pages" - API on tehniline ja ei ole kasutaja jaoks oluline.